contributor-guide/modules/ROOT/pages/requirements/library-metadata.adoc

```diff
@@ -3,7 +3,7 @@
 :idseparator: -
 :navtitle: Library Metadata
      
-A module can contain an optional file which describes the libraries that it contains. This is located at `meta/libraries.json`. It either contains a single json object for a single library, or a list or json objects for any number of libraries.
+A module must contain a file which describes the libraries that it contains. This is located at `meta/libraries.json`. It either contains a single json object for a single library, or a list or json objects for any number of libraries.
```

Comment:
What is a module? Please note that a module means a very specific thing in C++ and this might confuse people.

Reply:
Good point - added an explanation.